scdl/scdl.py

```diff
@@ -417,10 +418,7 @@ def download_track(track, playlist_name=None, playlist_file=None):
 
     title = track['title']
     title = title.encode('utf-8', 'ignore').decode('utf8')
-    if track['streamable']:
-        url = track['stream_url']
-    else:
-        logger.error('{0} is not streamable...'.format(title))
+    if not allow_download(track, title):
         return
```

Comment:
I don't know if `return if not allow_download(track, title)` would be simpler ?

Reply:
How do you mean? Delete the `allow_download()` function? Or combine lines 421 and 422 into `return if not allow_download(track, title)`

Reply:
combine lines 421 and 422 into `return if not allow_download(track, title)` :)

Reply:
done!